server/service/integration_mdm_profiles_test.go

```diff
@@ -2592,6 +2592,8 @@ func (s *integrationMDMTestSuite) TestHostMDMAppleProfilesStatus() {
 			{Identifier: mobileconfig.FleetCARootConfigPayloadIdentifier, OperationType: fleet.MDMOperationTypeInstall, Status: &fleet.MDMDeliveryVerifying},
 		},
 	})
+
+	// TODO(mna): integration test for exclude-any profiles (including VerifyHostMDMProfiles test) Win+Mac
```

Comment:
Just pointing out that the tests are not implemented yet, my plan is to add them next week but if we need to complete this earlier, integration tests would go here. The call to `VerifyHostMDMProfiles` is also needed, as there are changes to the queries of that method.